example-readme-template.md.txt

```diff
@@ -0,0 +1,93 @@
+
+<!-- Our articles have a specific structure. Learn more at CONTRIBUTING.md -->
+
+Introductory paragraph about the example that explains what problem it solves and why the reader should care. If possible, include use cases.
+
+In this tutorial, we will [configure/set up/build/deploy] [some thing]...
+
+When you're finished, you'll be able to...
+
+## Prerequisites
+
+<!-- The Prerequisites section includes an ordered list of required installation and configuration steps before the reader can deploy the Pulumi example. -->
+
+1. [Install Pulumi](https://www.pulumi.com/docs/reference/install/)
+1. Configure [cloud] <!-- We have setup pages for our popular cloud providers at https://www.pulumi.com/docs/reference/clouds/<cloud-provider>/setup/ -->
+1. Install [language runtime] <!-- We have setup pages for our supported language runtimes at https://www.pulumi.com/docs/intro/languages/<language>/ -->
+1. (Optional) List any other accounts needed, such as Twitter, Slack, or other services.
+
+<!-- Example:
+1. [Install Pulumi](https://www.pulumi.com/docs/reference/install/)
+2. [Configure your AWS Credentials](https://www.pulumi.com/docs/reference/clouds/aws/setup/)
+3. [Install Node.js](https://www.pulumi.com/docs/intro/languages/nodejs/)
+-->
+
+## Deploying the App
+
+### Step 1: Create a directory and cd into it
+
+For Pulumi examples, we typically start by creating a directory and changing into it. Then, we create a new Pulumi project from a template. For example, `azure-javascript`.
+
+First....
+
+Next...
+
+Finally...
+
+![Alt text for screen readers](/path/to/img.png) 
+
+<!--
+If showing a command, explain the command first by talking about what it does. Then show the command and its output. If the output is too long, you can clip it with `...` and only show the relevant parts. If your README includes images, make sure to include an alt text.
+
+Now transition to the next step by telling the reader what's next. If you're adding to a file, make sure to clearly explain that.
+-->
+
+
+### Step 2: Sentence case
+Another introduction
+
+Your content
+
+Transition to the next step.
+
+### Step 3
+
+Another introduction
+
+Your content
+
+Transition to the next step.
+
+## Clean Up
+
+<!--We generally ask the reader to run `pulumi destroy` and tear down the resources to avoid incurring any costs. -->
+
+
+## Summary
+
+In this article you [configured/set up/built/deployed] [something]. Now you can....
+
+<!-- Give a quick recap of what the readers have learned and optionally provide places for further exploration. -->
+
+## Next Steps
+
+<!-- Optionally include an unordered list of relevant Pulumi tutorials. -->
+
+<!-- Example:
+- [Create a load-balanced, hosted NGINX container service](https://www.pulumi.com/docs/tutorials/aws/ecs-fargate/)
+- [Create an EC2-based WebServer and associated infrastructure](https://www.pulumi.com/docs/tutorials/aws/ec2-webserver/)
+-->
+
```

Comment:
Not sure where, but I would add a link to a reference README file that we think is the best example.

Reply:
I would do this as a follow-up change after editing one of the featured tutorials: Now tracked as an item in https://github.com/pulumi/examples/issues/371.